rust/gui-client/build.sh

```diff
@@ -31,6 +31,11 @@ INTERMEDIATE_DIR=$(ls -d "$BUNDLES_DIR"/*/)
 cp src-tauri/deb_files/postinst src-tauri/deb_files/prerm "$INTERMEDIATE_DIR/control/"
 
 pushd "$INTERMEDIATE_DIR"
+# Substitute our own list of deps
+# This does nothing right now, but we'll need it for replacing Tauri
+sed '/^Depends:/c\Depends: libayatana-appindicator3-1, libwebkit2gtk-4.0-37, libgtk-3-0' control/control > control/control.new
```

Comment:
Can you not use `sed`'s "in-place" replacement?

Reply:
Yeah but I heard it's platform-specific and I didn't want to mess with it https://stackoverflow.com/a/12696224